docs/source/customization.rst

```diff
@@ -145,3 +148,71 @@ look for possible overrides using the name of the model itself, but also using
 all the parent models following the full inheritance chain. So if all your models
 inherit from ``MyAbstractModel``, you can for example create an override table
 for all your models by creating a ``myproject.tables.MyAbstractModelTable``.
+
+
+Extending autocomplete results
+------------------------------
+
+The autocomplete endpoint :py:class:`apis_core.generic.views.Autocomplete` can
+be extended to provide additional autocomplete results, that are not
+based on the default Django querysets. The autocomplete view always refers to a
```

Comment:
> The autocomplete view always refers to a Django model

It took me a while to figure out what that refers to. Given that *all* the views in the `generic.views` model use a model as a basis, I think this sentence can be removed (or moved to the top).

Reply:
ok, removed it